adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/DatasourceManager.java

```diff
@@ -75,6 +81,29 @@ public void deleteAllDatasources() {
     );
   }
 
+ public Datasource getParametrizedDatasource(Long id, DatasourceParameters datasourceParameters) {
+   Datasource datasource = getDatasource(id)
+     .orElseThrow(() -> new IllegalArgumentException("No datasource found with id " + id));
+   datasource.fillQueryParameters(datasourceParameters);
+   return datasource;
+ }
+
+ public DataBlob.MetaData trigger(Long id, DatasourceParameters datasourceParameters) {
```

Comment:
A unit test for this method would be nice

Reply:
We would only test mocks. Better test it in the integration and system test in a separate pr.